src/System.IO.Compression/src/System/IO/Compression/DeflateManaged/OutputWindow.cs

```diff
@@ -23,7 +23,7 @@ internal sealed class OutputWindow
 
         private readonly byte[] _window = new byte[WindowSize]; // The window is 2^18 bytes
         private int _end;       // this is the position to where we should write next byte
-        private int _bytesUsed; // The number of bytes in the output window which is not consumed.
+        internal int _bytesUsed; // The number of bytes in the output window which is not consumed.
```

Comment:
Rather than making this internal, did you consider either use the existing property or add a method that describes what it is you intend to do by setting it to 0? (eg: `Clear()` or something)

Reply:
There is no existing setter, but yes adding Clear() method instead is better and clear approach